satellite/console/consoleweb/utils.go

```diff
@@ -48,8 +51,13 @@ func getToken(req *http.Request) string {
 	return value[len(authorizationBearer):]
 }
 
+
 // getQuery retrieves graphql query from request
 func getQuery(req *http.Request) (query graphqlJSON, err error) {
+	if req.ContentLength > ContentLengthLimit {
```

Comment:
ContentLength header cannot be trusted.

Reply:
Good point. Replaced with ``http.MaxBytesReader`` where possible